src/css/gameboard.styl

```diff
@@ -101,15 +101,18 @@
 
     .counter
         padding: 0
-        font-size: 12px
-        width: 24px
-        height: 24px
-        line-height: 24px
+        font-size: 1rem
+        width: 2rem
+        height: 2rem
+        border-radius: 100%
+        line-height: 2rem
         text-align: center
-        margin: 2px
+        box-shadow: 
```

Comment:
This looks like an error.

Reply:
This may have been from the rebase, I'll amend.